app/views/maths_and_physics/claims/entire_term_contract.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for(:page_title, page_title(t("maths_and_physics.questions.has_entire_term_contract"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
```

Comment:
Do we need this? Won't the `current_claim` always be persisted at this point?

Reply:
Good spot, thanks. It's because I copied-and-pasted the template for the `teaching_maths_or_physics` question, which is a special case because it's the first question. I'll sort it out.